backend/src/test/test_authz_user.py

```diff
@@ -1,33 +1,34 @@
 from typing import Callable
 
 import pytest
+from sqlalchemy.orm import Session
 
 from app.core.authorization.authz_user import AuthzUser, ForbiddenError
 from app.core.data.crud import Crud
 from app.core.data.crud.code import crud_code
 from app.core.data.crud.project import crud_project
 from app.core.data.dto.user import UserRead
+from app.core.data.orm.code import CodeORM
 from app.core.data.orm.project import ProjectORM
 from app.core.db.sql_service import SQLService
 
 
 def test_assert_true(authz_user: AuthzUser):
-    authz_user.assert_bool(True, "")
+    authz_user.assert_condition(True, "")
 
     with pytest.raises(ForbiddenError):
-        authz_user.assert_bool(False, "")
+        authz_user.assert_condition(False, "")
 
 
 def test_assert_in_project(
-    user: int, project: int, session: SQLService, authz_user: AuthzUser
+    user: int, project: int, rollbacked_session: Session, authz_user: AuthzUser
 ):
```

Comment:
ich fände es besser statt user, project, code -> user_id, project_id, code_id zu benutzen (in allen tests).

Reply:
Hab mal ein issue für's refactoring gemacht :) https://github.com/uhh-lt/dwts/issues/324